script/Ciel/Day 10/10_01_CIEL10_3_1.txt

```diff
@@ -178,46 +190,47 @@ This is bad―――
 [sha:30527b52d9825f2259a04eb06fb979af83ac5931]{
 -- Page 157, Offset 27490.
 -- 背筋に走る警告が麻痺しつつある体に活を入れる。
-The warnings bells ringing through my spine breathe life into my catatonic body.
+The frantic warnings racing down my spine breathe life into my catatonic body.
 }
 [sha:52b866ffbf05505223a37fa6a2c015768e1f2ee0]{
 -- Page 157, Offset 27491.
 -- なんとか腰を起こし、出口のドアへと体を泳がせる。
-I somehow manage to half-rise to my feet, crawling toward the door.
+I somehow manage to get up off the floor and stumble toward the door.
 }
 [sha:89522f689db89610a022ea3f9128b2ebbf88bc13]{
 -- Page 158, Offset 27492.
 -- 「うーん、努力は買うけど残念賞～☆
 // I love her
-"Mmmm, I value your effort, but that will only get you the consolation prize～☆
+"Hmm, A for effort but I'm afraid you still get the consolation prize～☆
 }
 [sha:cb5cdb1cc1c101b133e342d97f26e9c8f4264d33]{
 -- Page 158, Offset 27493.
 -- 最後の食事が安物のケーキでゴメンね♥」
-Sorry that your last meal ended up being a cheap cake♥"
+Sorry that your last meal ended up being a slice of cheap cake ♥"
 }
 [sha:6e9202487601a7733a935bca7f452e656e4528c5]{
 -- Page 159, Offset 27494.
 -- 後頭部に重いものが落下する。
-Something heavy falls down onto the back of my head.
+Something heavy slams into the back of my head.
 }
 [sha:40e3c782f4a2418ec7d99f4cb74b2580cf3a37a0]{
 -- Page 159, Offset 27495.
 -- 手心は一切なかった。
-Not a single shred of mercy.
+She didn't show a single shred of mercy.
 }
 [sha:91d98a83def264be1d0e670373a9014a4360390c]{
 -- Page 159, Offset 27496.
 -- 耐えきれず断線する意識と、
-Unable to resist, a disconnecting consciousness―――
+Unable to withstand the blow, my consciousness blinks out.
 }
 [sha:887957f38af6d91e336aef4d386bb957b83ef123]{
 -- Page 159, Offset 27497.
 -- 朽ち木のように倒れる体。
-And a body collapsing like a putrid tree.
+My body collapses like a rotten tree.
 }
 [sha:775d5b667c40eaadc8a1239df2228d5ad186a65c]{
 -- Page 159, Offset 27498.
 -- この期に及んで危機感の欠けた学生と、日没の赤い影。
-A student who, at the very last moment, did not perceive the danger. And the red shadow of the setting sun.
+A student who failed to notice the danger they were in,
+ immersed in the red shadow of the setting sun.
```

Comment:
This and the previous two lines have the same structure...
That 

>My consciousness blinks out, unable to bear any more.
>And with it, my body collapses as if a rotted tree.
>A student lacking the good sense to understand the danger they were in until it was too late.
>And with him, the red shadow of the setting sun.

Last line is a bit irregular, but so is the Japanese.

Reply:
Yeah, I know, but it's a lot _more_ natural in JP than it is in EN. It does not strike me as a poetic upgrade to try and replicate the jp grammar here.

Reply:
It's not replicating the grammar so much as trying to give the same vibes.
As it is, no attempt was even made.

Reply:
Alright, how's this then?
```diff
-My consciousness blinks out, unable to bear any more.
+A fading consciousness, unable to bear any more.
 }
 [sha:887957f38af6d91e336aef4d386bb957b83ef123]{
 -- Page 159, Offset 27497.
 -- 朽ち木のように倒れる体。
-My body collapses like a rotten tree.
+Alongside it, a body collapsing like a rotten tree.
 }
 [sha:775d5b667c40eaadc8a1239df2228d5ad186a65c]{
 -- Page 159, Offset 27498.
 -- この期に及んで危機感の欠けた学生と、日没の赤い影。
-A student who failed to notice the danger they were in,
- immersed in the red shadow of the setting sun.
+A student who failed to notice the danger they were in.
+ Alongside them, the red shadow of the setting sun.
```

Reply:
Not good. 

I think ultimately you just need the repetition of と together with each line representing each subject. The verb can be added to この期に及んで危機感の欠けた学生, instead of the following line.

>My consciousness is fading, unable to bear any more.
>Alongside it, my body is collapsing, as if a rotten tree.

>There lies a student who only noticed the danger they were in when it was too late.
>Alongside him, the red shadow of the setting sun.

Reply:
I don't love that formulation because it's a split of verb sentences and noun sentences. I'd rather they all be or none be (like in the jp). The "as if a X" phrasing is also slightly overpretentious imo.

Reply:
Damn, that's the comparative phrase I use all the time...

I'm not sure if I'm entirely on-board with the all or nothing mentality. But anyway: I'm not sure if 'noun sentence' is what I'd call it, isn't it more that the first two have the possessive pronoun and the latter does not? In which case you'd rather change it to something like this: 

>My consciousness is fading, unable to bear any more.
Alongside it, my body is collapsing, as if a rotten tree.
There I lie, having only noticed the danger I was in when it was too late.
Alongside me, the red shadow of the setting sun.

Reply:
I meant noun sentences in that they're just kind of statements of existence, the verb is more of an adjective (a _fading_ consciousness, a _tree-falling-like_ body) than the focus of the sentence (consciousness _is fading_, body _is falling_). The pattern break is then 3-4, where the shadow of the setting sun isn't _doing_ something. I think we should probably also keep l3 a student vs 'I', since these bad end stanzas tend to be quite impersonal.

Reply:
Side note on the comparative phrase, the usage is what amplifies it - 
> it was quiet, as if a carpet of down had fallen over the world

not that weird, but ending a sentence with 'as if a $noun' is a phrasing that feels to me more common in older texts, or works attempting to emulate that style.

Reply:
I thought the sun not doing anything was kinda the point. It's the only clause in the Japanese that doesn't have a verb either.